lib/spack/spack/build_systems/r.py

```diff
@@ -25,6 +25,8 @@ class RPackage(PackageBase):
     """
     phases = ['install']
 
+    maintainers = ['glennpj']
```

Comment:
If a maintainer field is put in any specific `r-*` package then that will remove the maintainer value set here. I have not traced this but it seems that we want to be able to concatenate the `maintainers` list.

Reply:
Correct. Personally, I feel like if someone who actually develops a particular R packages wants to maintain it, they'll probably be better than you or I. You can always ask them to add you to the package-specific maintainers field. I don't see an easy way around this.